dev-site-config/generatedComponentConfig.js

```diff
@@ -372,10 +372,10 @@ import LightOverlay from '../packages/terra-overlay/examples/test-examples/Overl
 import OnRequestCloseOverlay from '../packages/terra-overlay/examples/test-examples/Overlay/OnRequestCloseOverlay.example';
 import DefaultOverlayContainer from '../packages/terra-overlay/examples/test-examples/OverlayContainer/DefaultOverlayContainer.example';
 import ProfileImageDefault from '../packages/terra-profile-image/tests/nightwatch/ProfileImageDefault';
-import ProgressBarGradient from '../packages/terra-progress-bar/tests/nightwatch/ProgressBarGradient';
-import ProgressBarColor from '../packages/terra-progress-bar/tests/nightwatch/components/ProgressBarColor';
-import ProgressBarDefault from '../packages/terra-progress-bar/tests/nightwatch/components/ProgressBarDefault';
-import ProgressBarSize from '../packages/terra-progress-bar/tests/nightwatch/components/ProgressBarSize';
+import ProgressBarGradient from '../packages/terra-progress-bar/examples/test-examples/ProgressBarGradient';
```

Comment:
If you rerun the generate-component config script, these components will not get imported. We need to update these moved examples to have `.example.jsx` extensions to be correctly added to the `generatedComponentConfig`